### ISSUE TITLE
How to change configuration to metric unit?

### ISSUE THREAD

**Issue by Andersants** [chrisk123999/chris-premades#259]
Hello, thank you for your module.
Is it possible to change the distances of spells and radius of effects to the metric unit? 
I'm GM of a BR group and we use the metric system to play.
I can't find a place to make this change.

**Comment by chrisk123999** [comment]
Not currently possible. It'll be looked at in the v12 update.

**Comment by Autumn225** [comment]
Metric is not well supported by the system and especially not by Midi-QOL, when possible, it will be added, but there is no ETA or guarantee at this time.

**Comment by Andersants** [comment]
Right, but is it possible to edit the scripts manually to set the distances? If so, where should I look?

**Comment by Andersants** [comment]
As a suggestion for improvement, perhaps it would be possible to make the distance of the effects search for the distance configured in the item itself or in the magic that the effect is linked to?

Currently, since I use the names of the items and spells translated into Portuguese, I drag the effects from the compendium directly onto the items.

**Comment by CaosFR** [comment]
Love to see this feature implement.

This subject is also discussed here, where I mention a script that converts feet into meters.

Casting a spell in feet is a “catastrophe” for metric users, as there is no conversion. This makes us cast areas 3.33 times too large ^^.

May this module can help to https://github.com/Roger92/metric-ruler-labels?tab=readme-ov-file#customizable-conversions  It is not changing any systems etc. It only applies an additional label to the UI.)
